public/500.html

```diff
@@ -13,119 +13,124 @@
-    <link rel="stylesheet" type="text/css" media="only screen and (max-width: 480px), handheld" href="/stylesheets/skins/skin_873_archive_2_0/4_site_maxwidth.handheld_.css">
-    <link rel="stylesheet" type="text/css" media="speech" href="/stylesheets/skins/skin_873_archive_2_0/5_site_speech_.css">
-    <link rel="stylesheet" type="text/css" media="print" href="/stylesheets/skins/skin_873_archive_2_0/6_site_print_.css">
-    <!--[if IE 8]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/7_site_screen_IE8_or_lower.css"><![endif]-->
-    <!--[if IE 5]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/8_site_screen_IE5.css"><![endif]-->
-    <!--[if IE 6]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/9_site_screen_IE6.css"><![endif]-->
-    <!--[if IE 7]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/10_site_screen_IE7.css"><![endif]-->  
+    <link rel="stylesheet" type="text/css" media="only screen and (max-width: 62em), handheld" href="/stylesheets/skins/skin_873_archive_2_0/4_site_midsize.handheld_.css">
+    <link rel="stylesheet" type="text/css" media="only screen and (max-width: 42em), handheld" href="/stylesheets/skins/skin_873_archive_2_0/5_site_narrow.handheld_.css">
+    <link rel="stylesheet" type="text/css" media="speech" href="/stylesheets/skins/skin_873_archive_2_0/6_site_speech_.css">
+    <link rel="stylesheet" type="text/css" media="print" href="/stylesheets/skins/skin_873_archive_2_0/7_site_print_.css">
+    <!--[if IE 8]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/8_site_screen_IE8_or_lower.css"><![endif]-->
+    <!--[if IE 5]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/9_site_screen_IE5.css"><![endif]-->
+    <!--[if IE 6]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/10_site_screen_IE6.css"><![endif]-->
+    <!--[if IE 7]><link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/skins/skin_873_archive_2_0/11_site_screen_IE7.css"><![endif]-->
+    <link href="/stylesheets/sandbox.css" media="screen" rel="stylesheet" type="text/css" />
   </head>
-	<body>
-		<ul id="skiplinks"><li><a href="#main">Main Content</a></li></ul>
-    <!-- BEGIN header -->
-    <div id="header" class="region">
-      <h1 class="heading">
-        <a href="/"><span>Archive of Our Own</span><sup> beta</sup><img alt="AO3 logo - the letters A O 3 combined with arms raised in celebration, symbolizing the joy of fannish creation in the Archive" class="logo" src="/images/ao3_logos/logo_42.png?1362864128" /></a> 
-      </h1>
+  <body>
+    <div id="outer" class="wrapper">
```

Comment:
FUNNY STORY. Somehow, the outer div we use on all the pages was missing from these, leading to chaos and mayhem once the new stylesheets were added. It is no longer missing.